### ISSUE TITLE
CFBundleVersion in info.plist

### ISSUE THREAD

**Issue by jackocnr** [phonegap/phonegap-cli#242]
TL;DR
So as far as I can see (please correct me if I'm wrong), the widget version attribute from my config.xml is used to populate the **version number** in my Android app, but the **build number** in my iOS app.

When I run `phonegap local build ios` it uses the `<widget>` version attribute from my config.xml to populate the `CFBundleVersion` in my app's info.plist file. According to Apple's [Information Property List Key Reference](https://developer.apple.com/library/ios/documentation/general/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html), `CFBundleVersion` should specify the "build version number" (AKA the build number), and `CFBundleShortVersionString` should specify the "release version number" (AKA the version).

This is inconsistent with the Android build process, which uses that widget version attribute to populate the `android:versionName`. According to Android's [Versioning Your Applications](http://developer.android.com/tools/publishing/versioning.html), `android:versionCode` should be "An integer value that represents the version of the application code, relative to other versions" (AKA the build number), and `android:versionName` should be "A string value that represents the release version of the application code, as it should be shown to users" (AKA the version).

Related issues:
https://github.com/phonegap/build/issues/229
http://community.phonegap.com/nitobi/topics/inability_to_specify_build_version_for_ios

**Comment by filmaj** [comment]
The latest cordova populates both fields. See https://github.com/apache/cordova-lib/blob/master/cordova-lib/src/cordova/metadata/ios_parser.js#L78-L81

Closing as no longer applicable.
